microbench/src/main/java/io/netty/microbench/handler/ssl/AbstractSslEngineBenchmark.java

```diff
@@ -83,11 +84,23 @@ private SslContext newServerContext() {
         }
 
         SSLEngine newClientEngine(String cipher) {
-            return configureEngine(clientContext.newEngine(PooledByteBufAllocator.DEFAULT), cipher);
+            return configureEngine(clientContext.newHandler(PooledByteBufAllocator.DEFAULT).engine(), cipher);
         }
 
         SSLEngine newServerEngine(String cipher) {
-            return configureEngine(serverContext.newEngine(PooledByteBufAllocator.DEFAULT), cipher);
+            return configureEngine(serverContext.newHandler(PooledByteBufAllocator.DEFAULT).engine(), cipher);
+        }
+
+        SslHandler newClientHandler(String cipher) {
+            SslHandler handler = clientContext.newHandler(PooledByteBufAllocator.DEFAULT);
+            configureEngine(handler.engine(), cipher);
+            return handler;
+        }
+
+        SslHandler newServerHandler(String cipher) {
+            SslHandler handler = serverContext.newHandler(PooledByteBufAllocator.DEFAULT);
```

Comment:
See my other comment. I think it would be better to create a new instance of the allocator each time.